client/src/core/components/load.vue

```diff
@@ -9,5 +9,9 @@ import Component from "vue-class-component";
 @Component
 export default class LoadComponent extends Vue {
```

Comment:
I think this component can be removed now entirely

Reply:
I've deleted the entire load.vue file in the latest commit.